Cargo.toml

```diff
@@ -1,9 +1,25 @@
 [workspace]
 members = [
   "packages/*",
-  "contracts/*",
+  "contracts/astroport-liquid-pooler",
+  "contracts/clock",
+  "contracts/clock-tester",
+  "contracts/ibc-forwarder",
+  "contracts/interchain-router",
+  "contracts/interchain-splitter",
+  "contracts/native-splitter",
+  "contracts/swap-covenant",
+  "contracts/swap-holder",
+  "contracts/two-party-pol-holder",
+  "contracts/two-party-pol-covenant",
+]
+exclude = [
```

Comment:
Have you tried to include "contracts/*" with the exclude? feels like it should work and it will exclude what you specified (very weird if its not, tho I can believe it)